script.js

```diff
@@ -1 +1,25 @@
+const elementButton = document.querySelector('button');
+let elementAddString = document.getElementById('addInput');
+let toDoList = document.getElementsByClassName('list')[0];
 
+elementButton.addEventListener('click', clickHandler); //добавляю событие на клик по кнопке
+function clickHandler (event) {
+    let newElement = document.createElement('div'); //создаю новый элемент, чтобы добавить его в список
+    newElement.classList.add('newElements'); //присваиваю класс новому элементу, чтобы изменять его свойства в CSS
+    
+    let textOfBusiness = document.createElement('div')
+    
+    let elementAddString = document.getElementById('addInput').value; //присваиваю новой переменной значение, которое ввел в поле ввода
+    if (elementAddString.lenth<0) return; //Проверка на то, что поле ввода не пустое
```

Comment:
свойства lenth нет, тут опечаточка
```suggestion
    if (elementAddString.length<0) return; //Проверка на то, что поле ввода не пустое
```

Reply:
ну и условие кажется не совсем правильным, меньше нуля длинна строки просто не может быть)